tests/cg_test.py

```diff
@@ -13,8 +13,10 @@
 # limitations under the License.
 
 import backend as T
+import autodiff as ad
```

Comment:
seems this line is redundent?

Reply:
This is passing in CI but not on my laptop.
I got 

```
ImportError while importing test module '/home/jiayu/workspace/AutoHOOT/tests/cg_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/cg_test.py:16: in <module>
    from utils import conjugate_gradient
utils.py:18: in <module>
    import autodiff as ad
autodiff.py:19: in <module>
    from utils import find_topo_sort, sum_node_list, inner_product, find_topo_sort_p
E   ImportError: cannot import name 'find_topo_sort' from 'utils' (/home/jiayu/workspace/AutoHOOT/utils.py)
================================================================== short test summary info =====================
```

I don't have a clear idea of why this is the case yet. Maybe you can double check, if it's just me, we can remove this in the PR and I will debug later.

Reply:
Yeah I don't have this problem, but seems you have already removed this line.